### DISCUSSION TITLE
Feature request : do not show boosts of followed accounts

### DISCUSSION THREAD

**Discussion by ArnonClark** [mastodon/mastodon#22947] [General]
Hi folks, I've been noticing that I end up seeing boosted toots of posts from people I already follow, which (absent QT functionality) seems like a waste of screenspace and resources. Might there be a way of checking boosted posts OP against followed users and not displaying this in the timeline? Maybe making it optional if there's a reason I'm not seeing for it, like covering bases to make sure one doesn't miss a popular post by one of your follows?
Thanks for everything you're doing!

**Comment by brownmichaelw** [comment]
This? In Preferences/Other:
[ x ] Group boosts in timelines (Recommended)
Do not show new boosts for posts that have been recently boosted (only affects newly-received boosts)

On that note though, there is considerable interest in an option to filter out all boosts from your timeline (or in a tab) so you only get a feed of things that those whom you follow actually author themselves.

**Reply by ArnonClark** [reply]
Yup. 🤦🏻‍♂️ That would be it.
